tool/plugin/lib/plugin.dart

```diff
@@ -657,20 +753,57 @@ class DeployCommand extends ProductCommand {
 /// Generate the plugin.xml from the plugin.xml.template file.
 /// If the --release argument is given, create a git branch and
 /// commit the new file to it, assuming the release checks pass.
-class GenCommand extends Command {
+/// Note: The product-matrix.json file includes a build spec for the
+/// EAP version at the end. When the EAP version is released that needs
+/// to be updated.
+class GenCommand extends ProductCommand {
   final BuildCommandRunner runner;
 
   GenCommand(this.runner);
 
   String get description =>
-      'Generate a valid plugin.xml and .travis.yaml for the Flutter plugin.\n'
+      'Generate a valid plugin.xml and .travis.yml for the Flutter plugin.\n'
```

Comment:
Just to confirm: we want the generated `.travis.yml` file committed to the repo (in order to have something to drive travis); the generated plugin.xml will be used for local development but likely shouldn't be committed?